pkg/cmd/pipelinerun/logs_test.go

```diff
@@ -1727,10 +1727,10 @@ func TestLogs_nologs_v1beta1(t *testing.T) {
 	}
 	prlo := logOpts(prName, ns, cs, dc, fake.Streamer([]fake.Log{}), false, false)
 	output, err := fetchLogs(prlo)
-	if err != nil {
-		t.Errorf("Unexpected error: %v", err)
+	if err == nil {
+		t.Errorf("Unexpected output: %v", output)
 	}
-	test.AssertOutput(t, "No logs found\n", output)
+	test.AssertOutput(t, "pipelinerun has not yet started", err.Error())
```

Comment:
Just one last nit then should be good!

```suggestion
	test.AssertOutput(t, "pipelinerun has not started yet", err.Error())
```